src/app/table-data/table-data.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { Account } from '../interfaces';
+import { Router } from '@angular/router';
+import { TableService } from '../table-service';
+@Component({
+  selector: 'app-table-data',
+  templateUrl: './table-data.component.html',
+  styleUrls: ['./table-data.component.scss'],
+})
+export class TableDataComponent implements OnInit {
+  accounts: Account[] = [];
+  id: number = 0;
```

Comment:
why this all we have as a field in this component?
id: number = 0;
 name: string = "";
  creationDate: Date | null = null;
  ownerName: string = "";
   ???